pyfolio/utils.py

```diff
@@ -598,3 +598,15 @@ def get_symbol_rets(symbol, start=None, end=None):
     return SETTINGS['returns_func'](symbol,
                                     start=start,
                                     end=end)
+
+
+def set_legend_location(ax):
+    """
+    Put legend in right of plot instead of overlapping with the it.
```

Comment:
Typo `instead of overlapping with the it.`

Reply:
fixed